frontend/src/container/AlertHistory/Statistics/TotalTriggeredCard/totalTriggeredCard.styles.scss

```diff
@@ -0,0 +1,3 @@
+.total-triggered-card {
+	width: 21.7%;
```

Comment:
Why specifically 21.7% ?

Reply:
I had calculated it. 
total width of the stats container= 1348 pixels
stats card width = 293 pixels. 

`293 * 100 / 1348 = 21.73% `

https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2819-7717&m=dev